src/global.css

```diff
@@ -3940,18 +3940,15 @@ div.ogi-movement .ogi-movement-target .ogi-movement-target-coords span,
 
 .smallplanet img {
   display: block;
-  height: 25px;
-  width: 25px;
-  top: 13px !important;
+  height: 25px !important;
+  width: 25px !important;
+  top: 17px !important;
+  left: 74px !important;
```

Comment:
!important It's necessary ? for height and with.

Reply:
Haven't checked this one, but between FF and Chrome, the internal CSS vs addon CSS is not in the same order. 
That mean you need to qualify more your selector if you don't want to use `!important`

Reply:
If we do not add it, in chrome (chromium), the properties seems to be not applied, because of what Blag says. But... as i can see now, not adding it the image gets the right size (25x25) anyway !? maybe it is the rest of the rules that let you assign that size.

On the other side, that is what happens in the current version of ogame and ogi, in chrome, exactly the same:

![imagen](https://github.com/user-attachments/assets/fd12c729-92f1-419b-9b34-8ed7917bddfa)

So... removing it